### PR TITLE
Edit My Name

### PR DESCRIPTION
**Pull request by andrewdudu**
- Add "Andrew" on README.md and README2.md

### CODE REVIEW

File: README4.md [hunk 1]
```diff
@@ -1 +1,4 @@
-new file 4
```
Comment (andirdju) [inline README4.md]: don't delete existing line

**Review by andirdju — CHANGES_REQUESTED** [review]

File: README3.md [hunk 2]
```diff
@@ -1 +1,4 @@
-this is my feature
+# git-training-2019-08-02
+henry
+Andrew
```
Comment (andirdju) [inline README3.md:2]: please use fullname

**Review by andirdju — CHANGES_REQUESTED** [review]

File: README5.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# git-training-2019-08-02
```
Comment (andirdju) [inline README5.md:1]: nice logic, very simple,

File: jwb.txt [hunk 4]
```diff
@@ -0,0 +1,6 @@
+1. Buat branch release
+2. Buat branch bug/b yang diambil dari branch release dan fix bug lalu commit
+3. Merge branch bug/b ke release
+4. Buat branch feature/a dan feature/c yang diambil dari branch release
+5. jika feature/a atau feature/c selesai, commit dan merge ke release
+6. Lakukan Pull Request ke branch master
```
Comment (andirdju) [inline jwb.txt]: pull request from which branch?
@andrewdudu

**Review by andirdju — CHANGES_REQUESTED** [review]
check comment

File: jwb.txt (hunk 4 above)
Comment (andrewdudu) [reply jwb.txt]: Pull Request dari branch release to master @andirdju

**Pull request closed without merging**